packages/prisma-client/prisma/migrations/template.txt

```diff
@@ -10,7 +10,10 @@ import { PrismaClient } from "./client";
 //   It's better to copy it to the migration directory.
 
 export default () => {
-  const client = new PrismaClient();
+  const client = new PrismaClient({
+    // Uncomment to see the queries in console as the migration runs
+    // log: ["query", "info", "warn", "error"],
```

Comment:
you can also do thi https://www.prisma.io/docs/concepts/components/prisma-client/debugging

Reply:
I tried this, but it doesn't output queries in SQL format, only in some prisma internal format